scripts/powershell/Office-365/GroupCreators.ps1

```diff
@@ -0,0 +1,27 @@
+﻿$GroupName = "Office 365 Group Creators"

```

Comment:
@mattco593 , can you add a header here?
For example: 
```
################################################
#
# Author: 
# Contact: github link or email if you want
#
# Title:
# Purpose:
#
#
################################################
```

Reply:
Yeah I will take care of that now.

Reply:
cool coo, just something generic, but to also give attribution and credit. :-)

Reply:
Done